src/Provider/Provider.Web/Orchestrators/Part1/LocationOrchestrator.cs

```diff
@@ -110,15 +111,15 @@ public async Task<VacancyEmployerInfoModel> GetVacancyEmployerInfoModelAsync(Vac
 
             var vacancy = await Utility.GetAuthorisedVacancyForEditAsync(_providerVacancyClient,
                 _recruitVacancyClient, locationEditModel, RouteNames.Location_Post);
-            var legalEntityId = employerInfoModel?.LegalEntityId ?? vacancy.LegalEntityId;
+            var accountLegalEntityPublicHashedId = employerInfoModel?.AccountLegalEntityPublicHashedId ?? vacancy.AccountLegalEntityPublicHashedId;
```

Comment:
Using the null-coalescing operator here will not check for empty strings - which seem important elsewhere?